components/Tabs/tests/unit/MDCTabBarLayoutTests.m

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <MDFInternationalization/MDFInternationalization.h>
+#import <XCTest/XCTest.h>
+
+#import "MDCItemBar.h"
+#import "MDCTabBar.h"
+
+@interface MDCTabBarLayoutTests : XCTestCase
+
+@end
+
+@implementation MDCTabBarLayoutTests
+
+- (void)testRTL {
+  // Create a tabbar with some dummy items and a dummy frame.
```

Comment:
Please add comments to break up this test into "Given", "When", and "Then" sections. That will be easier for others to read and will be consistent with our code base.

Blog post about this topic: https://martinfowler.com/bliki/GivenWhenThen.html
Example test with Given/When/Then: https://github.com/material-components/material-components-ios/blob/develop/components/BottomAppBar/tests/unit/BottomAppBarTests.m#L103

Reply:
Tried my best to split the tests into these sections while still making my intentions clear and understandable to a general reader.

PTAL.